DQMOffline/Trigger/src/EgHLTEleHLTFilterMon.cc

```diff
@@ -76,21 +86,22 @@ EleHLTFilterMon::EleHLTFilterMon(MonElemFuncs& monElemFuncs,const std::string& f
   diEleMassOnlyOneHighME_ = new DiEleMon(monElemFuncs.getIB(), filterName_+"_diEle_onlyOnePassFilter_massHigh",
 					 filterName_+"_diEle_onlyOnePassFilter Mass;M_{ee} (GeV/c^{2})",
 					 bins.massHigh.nr,bins.massHigh.min,bins.massHigh.max,&ParticlePair<OffEle>::mass);
-  
+ } 
 }
 
 EleHLTFilterMon::~EleHLTFilterMon()
 {
   for(auto & eleMonElem : eleMonElems_) delete eleMonElem;
   for(auto & eleFailMonElem : eleFailMonElems_) delete eleFailMonElem;
   for(auto & eleEffHist : eleEffHists_) delete eleEffHist;
+  if(!doHEP_){ 
```

Comment:
assuming the pointers are initialized (which they should be if not already), this if is not needed.